doc/Advanced.md

```diff
@@ -312,9 +313,14 @@ o2-qc --config json:/${QUALITYCONTROL_ROOT}/etc/basic.json --remote-batch result
 Please note, that the local batch QC workflow should not work on the same file at the same time.
 A semaphore mechanism is required if there is a risk they might be executed in parallel.
 
-To be done:
-- merging multiple files into one, to allow for cases, when local batch workflows cannot access the same file.
-- support for Post-Processing.
+The file is organized into directories named after 3-letter detector codes and sub-directories representing Monitor Object Collections for specific tasks.
+To browse the file, one needs the associated Quality Control environment loaded.
```

Comment:
Could you please elaborate a bit more about this? Does it mean checking the results for incomplete results? Maybe also add an example command?

Reply:
I will elaborate